main/options/options.cc

```diff
@@ -627,6 +627,8 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                                  "Packages which are allowed to ignore the restrictions set by `visible_to` "
                                  "and `export` directives",
                                  cxxopts::value<vector<string>>(), "<name>");
+    options.add_options(section)("packager-layer", "Valid layer names for packages.",
```

Comment:
Let's be sure to also mention here that the order matters, and what that order is.